plugins/premium/rtc.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: default
-title: RTC plugin
-title_nav: RTC (beta)
-description: Real time collaboration.
+title: Real-Time Collaboration plugin (beta)
+title_nav: Real-Time Collaboration (beta)
+description: The TinyMCE Real-Time Collaboration plugin
```

Comment:
None of our other premium plugins have descriptions like this, starting with `The TinyMCE <blah>` 🤔  I do see a few `<blah> for TinyMCE`, though 🤔

Reply:
```suggestion
description: The Real-Time Collaboration plugin for TinyMCE
```
?

Reply:
sure (I think I saw this in a couple of places, btw)